File: app/src/main/java/com/subhrajyoti/wallify/MyApplication.java
package com.subhrajyoti.wallify;


import android.app.Application;
import android.preference.PreferenceManager;

import org.polaric.colorful.Colorful;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        boolean dark = PreferenceManager.getDefaultSharedPreferences(getBaseContext()).getBoolean("dark", false);
        Colorful.config(getApplicationContext())
                .primaryColor(Colorful.ThemeColor.BLUE)
                .accentColor(Colorful.ThemeColor.PINK)
                .translucent(false)
                .dark(dark)
                .apply();

        Colorful.init(this);
    }
}

File: app/src/main/java/com/subhrajyoti/wallify/MyApplication.kt
package com.subhrajyoti.wallify

import android.app.Application
import android.content.Context
import android.preference.PreferenceManager
import org.polaric.colorful.Colorful
import org.polaric.colorful.Colorful.ThemeColor.BLUE
import org.polaric.colorful.Colorful.ThemeColor.PINK

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        context = applicationContext
        val dark = PreferenceManager.getDefaultSharedPreferences(baseContext).getBoolean("dark", false)
        Colorful.config(applicationContext)
                .primaryColor(BLUE)
                .accentColor(PINK)
                .translucent(false)
                .dark(dark)
                .apply()
        Colorful.init(this)
    }

    companion object {
        @JvmStatic
        lateinit var context: Context
            private set
    }
}